tests/test_CtfEstimate.py

```diff
@@ -18,10 +18,11 @@
 class CtfEstimatorTestCase(TestCase):
     def setUp(self):
         self.test_input_fn = "sample.mrc"
+        # Replaced test_output values after converting `estimate_ctf` units to angstroms.
         self.test_output = {
-            "defocus_u": 1.137359876e03,
-            "defocus_v": 9.617226108e02,
-            "defocus_ang": 1.5706205116381249,
+            "defocus_u": 10848.056020345417,  # replaced 1.137359876e03
```

Comment:
take out the `# replaced`. We can use `git` for that.

Reply:
Ok, thought you might suggest that. removing.